Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,5783 @@
+                                     outputs_key_field='id',
+                                     outputs=parsed_data,
+                                     raw_response=response)
+    return command_results
+
+
+def custom_predifined_whitelist_update_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Update a Custom Predifined member.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    id = args['id']
+    status = args['status']
+    rel_data = {}
+    # Get exist data
+    response = client.custom_predifined_whitelist_list_request()
+    data = response if client.version == ClientV1.API_VER else response['results']
+    for type in data:
+        for member in type['details']:
+            if member['value']:
+                id = member['id']
+                rel_data.update({f'{id}_enable': 'on'})
+    id = args['id']
+    if status == 'enable':
+        rel_data.update({f'{id}_enable': 'on'})
+    elif status == 'disable':
+        rel_data.pop(f'{id}_enable', None)
+    response = client.custom_predifined_whitelist_update_request(data=rel_data)
+
+    command_results = generate_simple_command_results('id', id, response, OutputTitles.CUSTOM_PREDIFINED_UPDATE)
+    return command_results
+
+
+def list_response_handler(client: Client,
+                          response: Union[List[Dict[str, Any]], Dict[str, Any]],
+                          data_parser: Callable,
+                          args: Dict[str, Any],
+                          sub_object_id: Optional[str] = None,
+                          internal_path: Optional[List[str]] = None,
+                          sub_object_key: str = '_id') -> Tuple[List[Dict[str, Any]], str, List[Dict[str, Any]]]:
+    """Handle the list output response to xsoar output.
+    Args:
+        client (Client): Fortiweb VM client.
+        response (Union[List[Dict[str, Any]], Dict[str, Any]]): Response from list request.
+        data_parser (_type_): Parser command.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+        sub_object_id (Optional[str]): Sub Object ID.
+        internal_path (Optional[List[str]]): Internal path inside the response.
+        sub_object_key (Optional[str]): Sub Object key.
+
+    Raises:
+        DemistoException: The object does not exist.
+
+    Returns:
+        Tuple[List[Dict[str, Any]], str, List[Dict[str, Any]]]: Filtered output to xsoar,
+        pagination message and response output.
+    """
+    if client.version == ClientV2.API_VER:
+        response = response['results']  # type: ignore # V2 always returns a Dict.
+    if internal_path:
+        response = dict_safe_get(response, internal_path)
+    elif not isinstance(response, list):
+        response_list = []
+        response_list.append(response)
+        response = response_list
```

Comment:
```suggestion
        response = [response_list]
```
Why not just like that?

Reply:
Because the response is a dictionary and when I do like that it becomes messy.

Reply:
Let's discuss about it tomorrow.